frontend/src/cards/MapCard.tsx

```diff
@@ -362,7 +388,9 @@ function MapCardWithKey(props: MapCardProps) {
                     data={
                       listExpanded
                         ? highestRatesList.concat(lowestRatesList)
-                        : dataForActiveBreakdownFilter
+                        : props.fips.isUsa()
+                        ? dataForActiveBreakdownFilter
+                        : dataForActiveBreakdownFilterWithSvi
```

Comment:
I don't super love nested ternaries.... if we do what i mentioned above and just keep the `dataForActiveBreakdownFilter` array as the real data, merging SVI into it as needed, we could avoid the nested ternary here. Otherwise, I think adding a comment might help for future confused code readers